src/playground-components/sentence-game.html

```diff
@@ -0,0 +1,89 @@
+            }
+          },
+					firstLetterChars: {
```

Comment:
Could you fix the indentation if this isn't on purpose?

Reply:
hahaha... Sorry, I was probably thinking that since it's a "playground," we could be messy with it.

Reply:
Or maybe not... It's not like that in my code. Maybe there's something I don't have in place here that I do with the bitbucket stuff.

Reply:
The reason why indentation would be important is because this will be a place a lot of new people would look at, and that would be confusing. But.. tabs vs spaces?!?!? no thank you